maps/tether/tether_defines.dm

```diff
@@ -155,7 +155,7 @@
 
 	lateload_z_levels = list(
 		list("Tether - Misc","Tether - Underdark","Tether - Plains"), //Stock Tether lateload maps
-		list("Offmap Ship - Talon Z1","Offmap Ship - Talon Z2"),
+		list("Offmap Ship - Talon V2")
```

Comment:
Needs comma at the end so it would compile

Reply:
no comma-ent